### ISSUE TITLE
Accessibility: Purple links on purple cards do not meet contrast guidelines

### ISSUE THREAD

**Issue by bridgetburke** [CSS-Tricks/serverless#34]
The site looks amazing and the attention to accessibility overall is great.

When there is a purple link (#9058b5) on the purple background (#37236b), the link doesn't meet WCAG guidelines contrast. https://serverless.css-tricks.com/ideas/

Thanks for the great content!

**Comment by chriscoyier** [comment]
fix going out
